deployments/chef/test/windows/Dockerfile.windows-2019

```diff
@@ -4,14 +4,15 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 
 RUN Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
 
-RUN choco install -y -f --no-progress chef-client
+RUN choco install -y -f --no-progress chef-client --version=17.6.18
```

Comment:
What's the significance of this version?

Reply:
There were compatibility issues with the latest versions for kitchen and chef-client, so I just pinned to known working versions for now.